examples/getting_started/5-Live_Data.ipynb

```diff
@@ -181,24 +195,18 @@
    "source": [
     "def interactive_clifford(a,b,c,d,x=0,y=0):\n",
     "    coords = clifford(a,b,c,d,x0=x,y0=y)\n",
-    "    return (hv.Curve(coords) * hv.Points(coords[0]) * hv.Curve(coords[:2], group='Init')\n",
-    "            * hv.Text(-0.75,1.35, 'x:{x:.2f} y:{y:.2f}'.format(x=coords[0][0],y=coords[0][1])))"
+    "    text = 'x:{x:.2f} y:{y:.2f}'.format(x=coords[0][0],y=coords[0][1])\n",
+    "    return (  hv.Points(coords)                                          # Attractor points\n",
+    "            * hv.Points(coords[0]).opts(color='black', size=10, alpha=1) # Starting point\n",
+    "            * hv.Curve(coords[:2], group='Init')                         # Line to first iteration\n",
+    "            * hv.Text(0,1.75, text))                                  # Starting position text"
```

Comment:
Have to say, I really dislike this indentation style, I know you disagree but I find it much more readable to put the ``*`` at the end of a line.

Reply:
Sure. I have no strong opinion but I think I have a mild preference for your suggestion.

Reply:
Looking at it, I did this for this case due to the comments at the end of each line. Changed it anyway in d9d8adee